### PR TITLE
fix: Add history entry when approving the slides.

### PR DESCRIPTION
**Pull request by kivinen**
Change os.rename to shutils.move as the submissions and proceedings are on the separate filesystems on the docker image, and this same thing might happen in the real environment in the future.

### CODE REVIEW

**Comment by kivinen** [comment]
This fixes #5432

**Review by jennifer-richards — APPROVED** [review]
This looks ok to me, though should there also be an event for disapproval?

**Comment by kivinen** [comment]
Jennifer Richards writes:
> @jennifer-richards approved this pull request.
> 
> This looks ok to me, though should there also be an event for disapproval?

For the initial presentation there is no way to log anything, as the
state is only created AFTER the document as been approved once.

Hmm... when the drafts are updated and then someone disapproves them,
then there would be possibility for making comment.
-- 
***@***.***

**Pull request merged**
